src/main/python/NBADataLoader.py

```diff
@@ -1,29 +0,0 @@
-from nba_api.stats.static import players
```

Comment:
Why was this file deleted? I think you may need to merge in the latest main. This file should not exist in main already due to a name and location change.

Reply:
yes forgot to merge in the latest main it was late.